.gitmodules

```diff
@@ -1,28 +1,30 @@
-[submodule "sorc/ufs_model.fd"]
-  path = sorc/ufs_model.fd
-  url = https://github.com/ufs-community/ufs-weather-model
-  ignore = dirty
 [submodule "sorc/wxflow"]
   path = sorc/wxflow
   url = https://github.com/NOAA-EMC/wxflow
-[submodule "sorc/gfs_utils.fd"]
-  path = sorc/gfs_utils.fd
-  url = https://github.com/NOAA-EMC/gfs-utils
-[submodule "sorc/ufs_utils.fd"]
-  path = sorc/ufs_utils.fd
-  url = https://github.com/ufs-community/UFS_UTILS.git
-[submodule "sorc/verif-global.fd"]
-  path = sorc/verif-global.fd
-  url = https://github.com/NOAA-EMC/EMC_verif-global.git
 [submodule "sorc/gsi_enkf.fd"]
   path = sorc/gsi_enkf.fd
   url = https://github.com/NOAA-EMC/GSI.git
 [submodule "sorc/gdas.cd"]
   path = sorc/gdas.cd
   url = https://github.com/NOAA-EMC/GDASApp.git
-[submodule "sorc/gsi_utils.fd"]
-  path = sorc/gsi_utils.fd
-  url = https://github.com/NOAA-EMC/GSI-Utils.git
+[submodule "sorc/verif-global.fd"]
```

Comment:
can the order and the indentation be reverted to `develop` so only the new submodule added (`upp`) is seen in the diff?

Reply:
I second this request.

Reply:
Done.

Reply:
This has not been done.

Reply:
In my [branch](https://github.com/HenryWinterbottom-NOAA/global-workflow/blob/feature/gwdev_issue_2329/.gitmodules):

I see the following:

```
[submodule "sorc/wxflow"]
  path = sorc/wxflow
  url = https://github.com/NOAA-EMC/wxflow
[submodule "sorc/gfs_utils.fd"]
  path = sorc/gfs_utils.fd
  url = https://github.com/NOAA-EMC/gfs-utils
[submodule "sorc/gsi_monitor.fd"]
  path = sorc/gsi_monitor.fd
  url = https://github.com/NOAA-EMC/GSI-Monitor
[submodule "sorc/ufs_utils.fd"]
  path = sorc/ufs_utils.fd
  url = https://github.com/ufs-community/UFS_UTILS
[submodule "sorc/gsi_utils.fd"]
  path = sorc/gsi_utils.fd
  url = https://github.com/NOAA-EMC/GSI-utils
[submodule "sorc/gsi_enkf.fd"]
  path = sorc/gsi_enkf.fd
  url = https://github.com/NOAA-EMC/GSI.git
[submodule "sorc/verif-global.fd"]
  path = sorc/verif-global.fd
  url = https://github.com/NOAA-EMC/EMC_verif-global.git
[submodule "sorc/gdas.cd"]
  path = sorc/gdas.cd
  url = https://github.com/NOAA-EMC/GDASApp.git
[submodule "sorc/ufs_model.fd"]
  path = sorc/ufs_model.fd
  url = https://github.com/ufs-community/ufs-weather-model
[submodule "sorc/upp.fd"]
  path = sorc/upp.fd
  url = https://github.com/NOAA-EMC/UPP
```

Reply:
That is not the order they appear in develop: https://github.com/NOAA-EMC/global-workflow/blob/develop/.gitmodules